PennMobile/GSR-Booking/Controllers/GSRGroups/GSRGroupInviteViewController.swift

```diff
@@ -0,0 +1,286 @@
+//
+//  GSRGroupInviteViewController.swift
+//  PennMobile
+//
+//  Created by Lucy Yuewei Yuan on 11/3/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+//Users/lucyyyw/Desktop/pennlabs/penn-mobile-ios/PennMobile/GSR-Booking/Controllers/GSRLocationsController.swift
+import UIKit
+
+class GSRGroupInviteViewController: UIViewController {
+    
+    fileprivate var dummyLabel: UILabel!
+    fileprivate var closeButton: UIButton!
+    fileprivate var inviteUsersLabel: UILabel!
+    fileprivate var searchBar: UISearchBar!
+    fileprivate var sendInvitesButton: UIButton!
+    fileprivate var tableView: UITableView!
+    
+    fileprivate let disabledBtnColor = UIColor(red:32/255.0, green:156/255.0, blue:238/255.0, alpha:0.5)
```

Comment:
Can we switch these to Color instances? Trying to remove all custom colors from the app, even if its a lil different from my wireframes